docs/ref/reference.rst

```diff
@@ -85,225 +113,242 @@ nodes of the formation.
+
+  3. If the reverse DNS lookup is successful , then `pg_autoctl` does with a
+     forward DNS lookup of that hostname.
+
+When the forward DNS lookup response in step 3. is an IP address found in
+one of our local network interfaces, then `pg_autoctl` uses the hostname
+found in step 2. as the default `--hostname`. Otherwise it uses the IP
+address found in step 1.
+
+You may use the `--hostname` command line option to bypass the whole DNS
+lookup based process and force the local node name to a fixed value.
+
+The ``--auth`` option allows setting up authentication method to be used for
+connections from data nodes with ``autoctl_node`` user. When testing
+pg_auto_failover for the first time using ``--auth trust`` makes things
+easier. When getting production ready, review your options here and choose
+at least ``--auth scram-sha-256`` and make sure password is manually set on
+the monitor, and appropriate setting is added to `.pgpass` file on data
+node. You could also use some of the advanced Postgres authentication
+mechanism such as SSL certificates.
 
-       3. If the reverse DNS lookup is successful , then `pg_autoctl` does
-          with a forward DNS lookup of that hostname.
-
-     When the forward DNS lookup response in step 3. is an IP address found
-     in one of our local network interfaces, then `pg_autoctl` uses the
-     hostname found in step 2. as the default `--hostname`. Otherwise it
-     uses the IP address found in step 1.
-
-     You may use the `--hostname` command line option to bypass the whole
-     DNS lookup based process and force the local node name to a fixed
-     value.
-
-     The ``--auth`` option allows setting up authentication method to be
-     used for connections from data nodes with ``autoctl_node`` user. When
-     testing pg_auto_failover for the first time using ``--auth trust``
-     makes things easier. When getting production ready, review your options
-     here and choose at least ``--auth scram-sha-256`` and make sure
-     password is manually set on the monitor, and appropriate setting is
-     added to `.pgpass` file on data node. You could also use some of the
-     advanced Postgres authentication mechanism such as SSL certificates.
-
-     See :ref:`pg_auto_failover_security` for notes on `.pgpass`
-
-  - ``pg_autoctl run``
-
-    This command makes sure that the PostgreSQL instance for the monitor is
-    running, then connects to it and listens to the monitor notifications,
-    displaying them as log messages::
+See :ref:`pg_auto_failover_security` for notes on `.pgpass`
 
-      $ pg_autoctl run --help
-      pg_autoctl run: Run the pg_autoctl service (monitor or keeper)
-      usage: pg_autoctl run  [ --pgdata ]
+pg_autoctl run
+^^^^^^^^^^^^^^
 
-        --pgdata      path to data directory
+This command makes sure that the PostgreSQL instance for the monitor is
+running, then connects to it and listens to the monitor notifications,
+displaying them as log messages::
```

Comment:
This seems a bit outdated with the new subprocess/service architecture. Also maybe it should include something about the keeper too (not sure since this is in the monitor section).

Reply:
I have updated the text to match the current situation better.